### ISSUE TITLE
Multi-line description truncated in summary

### ISSUE THREAD

**Issue by neothemachine** [esdoc/esdoc#85]
When I write a description sentence in two lines esdoc just uses the first line in the summary which is really odd. It should use everything until a blank line or `@something` comes.

**Comment by h13i32maru** [comment]
@neothemachine Thanks for this issue!
Only first line is exist in summary, this is spec of ESDoc. The spec is inspired from [Android Document](http://developer.android.com/reference/android/app/Activity.html).

And for now, I do not have plan to support ``@summary`` tag.

**Comment by neothemachine** [comment]
I think the android docs actually use the first *sentence*, not the first line.

Why should I violate a given max line length (let's say 80 chars) just because my first description sentence is 100 chars? It is natural to break it to two lines in that case. Please rethink it. I realize it is not a bug but a feature request.

**Comment by h13i32maru** [comment]
Oh, sorry, I was mistake.

```javascript
/**
 * This is awesome function
 * and this is super cool function.
 */
function foo(){}
```
Currently, summary of this case is ``This is awesome function``. But do you want that summary is ``This is awesome function and this is super cool function.``?

If this is your request, I will adopt the request.

**Comment by neothemachine** [comment]
Exactly, but be careful with the details. It may happen that the dot at the end of the sentence is missing. So you also need to cut off if there's a blank line coming or some tags start. All the examples below should work as you would expect:

```js
/**
 * Ideally, this is all in one line.
 */
function foo(){}
```

```js
/**
 * Ideally, this is all in one line, but sometimes it is just
 * a bit longer.
 */
function foo(){}
```

```js
/**
 * Ideally, this is all in one line, but sometimes it is just
 * a bit longer
 * 
 * Here comes more detailed descriptions. Note I forgot the
 * dot above.
 */
function foo(){}
```

```js
/**
 * Ideally, this is all in one line, but sometimes it is just
 * a bit longer
 * @return {string}
 */
function foo(){}
```

**Comment by h13i32maru** [comment]
Thank you. Please wait a moment. I try to fix it.

**Comment by jdfreder** [comment]
:+1:

**Comment by h13i32maru** [comment]
@neothemachine @jdfreder I published v0.3.1 that fixed this issue.
